app/services/message-service.js

```diff
@@ -31,24 +31,31 @@ export const sendMessageToSingleFrontDoor = async (
 export const buildOutboundMessage = (messageId, inboundMessage) => {
   const service = SOURCE_SYSTEM
 
-  return {
+  const outboundMessage = {
     id: messageId,
     source: service,
     specversion: '1.0.2',
     type: 'uk.gov.ffc.ahwr.comms.request',
     datacontenttype: 'application/json',
-    time: inboundMessage.dateTime.toString(),
+    time: inboundMessage?.dateTime?.toString(),
```

Comment:
Interesting that inbound message can be null/undefined here. When/how could this ever happen?